doc.go

```diff
@@ -0,0 +1,6 @@
+/*
+Package gotwtr provides a Client for the Twitter V2 API.
+
+This library covers all Twitter API v2 method.
```

Comment:
I suggest to describe more detail of this library.
Because, we've not covered all Twitter V2 API, yet.
We covers only Twitter V2 API supported by OAuth 2.0 Bearer Token.
This bit different may cause someone who uses this library misunderstanding.
That's why, I think it better to be more clearly.

Reply:
>This bit different may cause someone who uses this library misunderstanding.
That's why, I think it better to be more clearly.

That's correct, so I'll fix it.

Reply:
@sivchari 
Please review doc comment.